gtsherman/generic-cross-validation
# Reorganize for install via setup.py

Not sure if you'd be interested in this.  I've been porting much of my temporal IR work to a Python-centric framework (based around [pyndri](https://github.com/cvangysel/pyndri/) and [parsl](https://github.com/Parsl/parsl), among other things.  I wanted to use your CV work and thought it might be handy to package for installation via setup.py or eventually pip. I made the bold assumption that this would be in the very stale "retrievable" namespace we once discussed...

In short:
* Moved modules into retrievable/cv subdirectory
* Added setup.py including console script
* Added a placeholder license, because might want one.

To test:
`pip install -e .` or `python setup.py install`

then
`run-cv ...`

And we should chat someday...
